build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ArtifactoryClientConfiguration.java

```diff
@@ -141,6 +153,8 @@
  * @author freds
  */
 public class ArtifactoryClientConfiguration {
+    private static final String ALGORITHM = "AES";
```

Comment:
Let's export all encyption logic functions to a dedicated "EncryptionUtils" class